docker-compose-dev.yml

```diff
@@ -25,8 +25,8 @@ services:
     ports:
      - "${WDQS_FRONTEND_PORT}:80"
     restart: ${RESTART}
-    environment:
-      - WIKIBASE_HOST=wikibase.svc # ? see https://github.com/MaRDI4NFDI/portal-compose/issues/35
```

Comment:
Please make sure the wdqs frontend service depends on wikibase. This could be best done in docker-compose.yml. Otherwise the wdqs-frontend service could start before the wikibase service and crash.